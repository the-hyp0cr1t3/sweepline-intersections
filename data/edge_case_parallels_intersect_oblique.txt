3
2 3 7 6
3 6 6 2
3 7 6 3
