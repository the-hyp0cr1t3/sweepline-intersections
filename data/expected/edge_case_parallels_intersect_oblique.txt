2
4.2413793103 4.3448275862 1 2
4.7586206897 4.6551724138 1 3

